.gitignore

```diff
@@ -4,4 +4,5 @@ screenshots
 .ruby-gemset
 tags
 .bundle
-.wercker/
\ No newline at end of file
+.wercker/
+.idea/
```

Comment:
get you editor to add new lines automatically (and also strip trailing white space if you already haven't)

Reply:
@StupidCodeFactory  Yep just noticed this. I have a new Rubymine installation and it screwed up the formatting. I'll fix.
Could be worth adding CodeClimate / linting in another PR